src/dscope.d

```diff
@@ -569,14 +569,30 @@ struct Scope
      */
     extern (C++) static void deprecation10378(Loc loc, Dsymbol sold, Dsymbol snew)
     {
+        // Bugzilla 15857
```

Comment:
Well, this is how we did in the rest of the code base. Sure someone could convert all of them to links if that's deemed helpful, but let's not start with a single one at a random PR.